numpy/_core/src/common/npy_svml.h

```diff
@@ -1,3 +1,27 @@
+#if NPY_SIMD && defined(NPY_HAVE_AVX512_SPR) && defined(NPY_CAN_LINK_SVML)
+extern __m512h __svml_exps32(const npy_half*, npy_half*, npy_intp);
```

Comment:
```suggestion
extern void __svml_exps32(const npy_half*, npy_half*, npy_intp);
```
The signature here is a bit weird. any explanation?

Reply:
oh, yikes. That is incorrect, surprising the compiler doesn't catch that.

Reply:
Fixed it.